Plugins~/Src/MeshSyncClient/Include/MeshSyncClient/msTransformManager.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "pch.h"
+#include "MeshSync/MeshSync.h"
+
+namespace ms{
+	class TransformManager {
```

Comment:
Can you add some documentation for this class?

Reply:
Would be nice to have some for the functions as well

Reply:
Added XML comments to class and interface